js/about/passwords.js

```diff
@@ -23,13 +66,17 @@ class SiteItem extends React.Component {
   }
 
   render () {
-    return <tr className='passwordItem'>
-      <td className='passwordActions'>
-        <span className='passwordAction fa fa-times' title='Remove site'
-          onClick={this.onDelete} />
-      </td>
-      <td className='passwordOrigin'>{this.props.site}</td>
-    </tr>
+    return <PasswordsTr data-test-id='passwordItem'>
+      <ActionsTd>
+        <span className={css(styles.passwordAction)}>
+          <span className='fa fa-times' title='Remove site'
```

Comment:
> can we go ahead on this and localize title?

I noticed but did not fix as I thought it is another issue. It should be handled with a follow up task.